examples/PowerElectronics/Microgrid/Microgrid.cpp

```diff
@@ -315,20 +316,21 @@ int main(int argc, char const* argv[])
   sysmodel->initialize();
   sysmodel->evaluateResidual();
 
-  std::vector<double>& fres = sysmodel->getResidual();
-  std::cout << "Verify Intial Resisdual is Zero: {\n";
-  for (size_t i = 0; i < fres.size(); i++)
-  {
-    printf("%lu : %e \n", i, fres[i]);
-  }
-  std::cout << "}\n";
+  // // Optional debugging output
```

Comment:
Consider adding a verbosity flag and a conditional to get to this code. Or remove completely.